sphinx/util/logging.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.util.logging
+    ~~~~~~~~~~~~~~~~~~~
+
+    Logging utility functions for Sphinx.
+
+    :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+
+def is_suppressed_warning(type, subtype, suppress_warnings):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Redefining built-in 'type'](https://www.codacy.com/app/georg/sphinx/file/2137828571/issues/source?bid=3149936&fileBranchId=3149940#l13)